langchain/chat_models/openai.py

```diff
@@ -256,6 +248,10 @@ def _default_params(self) -> Dict[str, Any]:
             "stream": self.streaming,
             "n": self.n,
             "temperature": self.temperature,
+            "api_key": self.openai_api_key,
+            "api_base": self.openai_api_base,
+            "organization": self.openai_organization,
+            # "proxy": {"http": self.openai_proxy, "https": self.openai_proxy}
```

Comment:
why not proxy?

Reply:
> why not proxy?

We need it, but I haven't found a feasible way to write it.